### PR TITLE
[W5][W11-2]Ren Ying

### PR DESCRIPTION
**Pull request by hiyingnn**

### CODE REVIEW

File: src/seedu/addressbook/parser/Parser.java [hunk 1]
```diff
@@ -11,17 +11,7 @@
...
-import seedu.addressbook.commands.Command;
-import seedu.addressbook.commands.DeleteCommand;
-import seedu.addressbook.commands.ExitCommand;
-import seedu.addressbook.commands.FindCommand;
-import seedu.addressbook.commands.HelpCommand;
-import seedu.addressbook.commands.IncorrectCommand;
-import seedu.addressbook.commands.ListCommand;
-import seedu.addressbook.commands.ViewAllCommand;
-import seedu.addressbook.commands.ViewCommand;
+import seedu.addressbook.commands.*;
```
Comment (limli) [inline src/seedu/addressbook/parser/Parser.java:14]: try not to use *

File: test/java/seedu/addressbook/commands/SortCommandTest.java [hunk 2]
```diff
@@ -0,0 +1,21 @@
+package seedu.addressbook.commands;
+
+import org.junit.Test;
+import seedu.addressbook.data.AddressBook;
+import seedu.addressbook.util.SortedPersons;
+import seedu.addressbook.util.TypicalPersonsVariedCase;
+
+
+import static org.junit.Assert.*;
```
Comment (limli) [inline test/java/seedu/addressbook/commands/SortCommandTest.java]: try not to use *

File: src/seedu/addressbook/data/person/UniquePersonList.java [hunk 3]
```diff
@@ -129,15 +132,32 @@ public void clear() {
...
+    }
+
+
     @Override
     public Iterator<Person> iterator() {
         return internalList.iterator();
     }
 
+    @Override
+    public void forEach(Consumer<? super Person> action) {
```
Comment (limli) [inline src/seedu/addressbook/data/person/UniquePersonList.java]: why override this to be empty?

File: src/seedu/addressbook/parser/Parser.java [hunk 4]
```diff
@@ -72,34 +62,36 @@ public Command parseCommand(String userInput) {
         final String arguments = matcher.group("arguments");
 
         switch (commandWord) {
+            case AddCommand.COMMAND_WORD:
```
Comment (limli) [inline src/seedu/addressbook/parser/Parser.java:65]: case should not have indentation

**Comment by sinhayash** [comment]
Good effort. You can improve upon:
1. Nice that you have updated tests.
2. It is good that you have incremental changes. You can try to have commits done in a batch for each enhancement. 
3. Header comment missing for class `TypicalPersonsVariedCase`. Make sure you don't miss it for other classes as well.
After reading the comments, please close the PR.

**Pull request closed without merging**